src/components/Stream/Stream.tsx

```diff
@@ -145,13 +147,29 @@ class Stream extends Component<IStreamProps, IStreamState> {
                 } else {
                     await signalRManager.sendCommand("Resume");
                 this.props.onChangeStreamingStatus(true);
-                playbackManager.resume(this.props.onChangeCurrentPlaybackTime, this.pause.bind(this));
+                playbackManager.resume(this.props.onChangeCurrentPlaybackTime, this.reset.bind(this));
                 this.props.onChangePaused(false);
                 console.log(this.props.currentSpeechId + " and id " + id);
                 console.log(this.props.isPlaying);
                 }              
             } else if (this.props.isPlaying) {
                 await this.pause();
+            } else if (this.props.paused && this.props.currentSpeechId !== id) {
```

Comment:
What purpose does it have ?
Will this condition ever happen ?
Should be removed if it is unnecessary.